includes/paypro/wc/settings-page.php

```diff
@@ -35,16 +35,17 @@ public function output() {
                 submit_button('Create a webhook', 'secondary', 'save');
             } else {
                 try {
-                    $webhook = PayPro_WC_Plugin::$paypro_api->getWebhook($webhook_id);
+                    $webhook      = PayPro_WC_Plugin::$paypro_api->getWebhook($webhook_id);
+                    $webhook_name = $webhook ? $webhook->name : null;
```

Comment:
It would be nice to render `submit_button('Create a webhook', 'secondary', 'save');`, when the webhook is not found